### PR TITLE
implemented table.regions() function  which will returns list of region of this table   

### LINKED ISSUE #29: Implement Table.regions()
The Java client implements Table.regions() using the SampleRowKeys RPC to get the start/stop keys.  The happybase client should do so as well.

### PR DESCRIPTION
**Pull request by pchauhan-qlogic**
Fixes #29

### CODE REVIEW

**Comment by pchauhan-qlogic** [comment]
Implemented table(regions)
unit tests
system test

**Comment by AVaksman** [comment]
@tseaver can you take a look at this please.

File: src/google/cloud/happybase/connection.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def tables(self):
 
         return table_names
 
-    def create_table(self, name, families):
+    def create_table(self, name, families, initial_split_keys=[]):
```
Comment (sduskis) [inline src/google/cloud/happybase/connection.py]: You can't change this interface.  Please revert the use of `initial_split_keys`.  If you need split keys, use the underlying google-cloud-python connection instead.

File: src/google/cloud/happybase/region_locator.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Bigtable HappyBase table module."""
+
+
+class RegionLocation(object):
```
Comment (sduskis) [inline src/google/cloud/happybase/region_locator.py]: Please remove this.

File: src/google/cloud/happybase/table.py [hunk 3]
```diff
@@ -142,20 +142,23 @@ def families(self):
 
     def regions(self):
```
Comment (sduskis) [inline src/google/cloud/happybase/table.py]: The signature should match the original happybase implementation.  Please see here: https://github.com/wbolster/happybase/blob/master/happybase/table.py#L76

**Pull request closed without merging**